src/components/Common/OrganizationCard.js

```diff
@@ -84,15 +84,16 @@ class OrganizationCard extends Component {
             name, categoryautosortscript, overview, location, website, facebookUrl,
             instagramUrl, twitterUrl, phone
         } = this.props.organization;
-        let distance, distanceElement;
+        let distance, distanceElement, distanceURL;
         let url = this.validatedUrl(website);
-        if (this.props.currentPos) {
+        if (this.props.currentPos && this.props.organization.coordinates) {
             distance = getDistance({coordinates: this.props.organization.coordinates}, this.props.currentPos);
             if (distance) {
                 distanceElement = <p>Distance from your Location: {distance} miles</p>
             }
-        }
-
+            distanceURL = "https://www.google.com/maps?saddr=My+Location&daddr="+
+                this.props.organization.coordinates.lat+","+this.props.organization.coordinates.lng;            
```

Comment:
Quick thing - chrome/google generally handle these things gracefully, but be aware there are some characters that are reserved/unsafe for URLs

https://www.google.com/search?q=url+unsafe+characters

can be fixed with encoding- https://www.w3schools.com/tags/ref_urlencode.asp

Reply:
I've encoded the coordinates. I've made the changes on behalf of @sayansaha1143 since his git doesn't work.